nilearn/datasets/tests/test_atlas.py

```diff
@@ -16,76 +15,12 @@
 
 from nilearn._utils import data_gen
 from nilearn._utils.testing import serialize_niimg
-from nilearn.datasets import atlas, utils
+from nilearn.datasets import atlas
 from nilearn.datasets._testing import dict_to_archive
+from nilearn.datasets.utils import _fetch_files
 from nilearn.image import get_data
 
 
-def test_get_dataset_dir(tmp_path):
```

Comment:
So, this test is no longer necessary ?

Reply:
As you guessed this test was duplicated: see [here](https://github.com/nilearn/nilearn/issues/3570#issuecomment-1677562849)